native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -52,10 +53,17 @@ class ViewController: UIViewController {
         let nib = UINib(nibName: "JourneyCell", bundle: nil)
         resultTable.register(nib, forCellReuseIdentifier: customCellIdentifier)
         view.bringSubviewToFront(loadingView)
+        
+        submitButton.setTitle("Reload Stations", for: .normal)
```

Comment:
This wasn't what I had in mind, but sure, this prob works better instead of adding another button.